assets/js/incentiveLevel.js

```diff
@@ -1,13 +1,15 @@
 $(function () {
+
   // record incentive level confirmation page
   $('.govuk-button').on('click', function (e) {
     const $prompt = $(e.target)
-    const gaId = $prompt.data('ga-id')
-    const caseload = $prompt.data('case-load')
-    if (gaId && typeof gtag === 'function') {
-      gtag('event', 'click', {
-        event_category: gaId,
-        event_label: caseload,
+    const gaCategory = $prompt.data('ga-category')
+    const gaAction = $prompt.data('ga-action')
+
+    if (gaCategory && typeof gtag === 'function') {
+      gtag('event', 'incentives_event', {
+        category: gaCategory,
+        action: gaAction,
       })
```

Comment:
I think `gaSendEvent` would do pretty much the same thing, but I haven't touched this in awhile so double-check: https://github.com/ministryofjustice/hmpps-incentives-ui/blob/9f8be5b8ff32678665b4ecbccbc612e34c80200b/assets/js/gaSendEvents.js

Reply:
you may be able to simplify with something along the lines of `$('.govuk-button').on('click', gaSendEvent)`

Reply:
`gaSendEvent` is not loaded on this page. It _could_ be but it has separate initialisation which would need a tweak. I thought at this point the two could be separate – we don't have any nice front end JS bundling in our projects